### PR TITLE
add a CLI to g11n-pipeline

### LINKED ISSUE #127: add CLI
Add a basic CLI, to be installed with the same name as the package (`g11n-pipeline`). 

Follow the [Java gp-cli](https://github.com/IBM-Cloud/gp-java-tools/blob/master/gp-cli/README.md) as an example, but emphasize features currently missing from the Java CLI.

### PR DESCRIPTION
**Pull request by srl295**
Fixes #127

### CODE REVIEW

**Comment by srl295** [comment]
- [ ] support env vars for credentials

File: bin/gp-js-cli [hunk 1]
```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+/*
+ * Copyright IBM Corp. 2015-2017
```
Comment (jugu) [inline bin/gp-js-cli]: copy right year 2018?

File: lib/client.js [hunk 2]
```diff
@@ -557,6 +557,36 @@ class Client {
...
+   *
+   * @param {Object} [opts={}] - optional map of options
+   * @param {TranslationRequest~getTranslationRequestsCallback} cb - callback yielding a map of Translation Requests
+   */
+  docTrs(opts, cb) {
+    opts = utils.fixArgs(arguments);
+    if ((cb = arguments[1])) {
+      return utils.depromise(this.docTrs(opts), cb);
+    }
+    var serviceInstance = this.getServiceInstance(opts);
```
Comment (jugu) [inline lib/client.js]: no linter? Using `var`?

File: lib/gpcli-verbs.js [hunk 3]
```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright IBM Corp. 2015-2017
```
Comment (jugu) [inline lib/gpcli-verbs.js]: Copyright year 2018?

File: bin/gp-js-cli (hunk 1 above)
Comment (srl295) [reply bin/gp-js-cli]: will fix

File: lib/gpcli-verbs.js (hunk 3 above)
Comment (srl295) [reply lib/gpcli-verbs.js]: will fix

File: lib/client.js (hunk 2 above)
Comment (srl295) [reply lib/client.js]: linter, but lots of `var` still. Fixed this instance.

**Pull request merged**